### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by goderbauer** [comment]
Not sure if useful, but seems related: Looks like GitHub now has a more advanced issue template manager that would even allow us to show different templates to users based on what they want to do: https://help.github.com/articles/about-issue-and-pull-request-templates/

**Comment by Hixie** [comment]
Oh that's great. Let me move this to that.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,18 +1,60 @@
...
+-->
+
 ## Steps to Reproduce
 
-Please tell us what you were doing and what went wrong. If you are running flutter tools from the command line, please try adding the `-v` or `--verbose` option to gather more information.
+<!--
+
+     Please tell us exactly how to reproduce the problem you are running into.
+
+     Please attach a small application (ideally just one main.dart file) that
```
Comment (tvolkert) [inline ISSUE_TEMPLATE.md]: Maybe inform them of gist.github.com?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,18 +1,60 @@
...
+     If you are looking for support, please check out our documentation
+     or consider asking a question on Stack Overflow:
+      * https://flutter.io/
+      * https://docs.flutter.io/
+      * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+
+     If you have found a bug or if our documentation doesn't have an answer
+     to what you're looking for, then fill our the template below. Please read
+     our guide to filing a bug first:  https://flutter.io/bug-reports/
+
```
Comment (tvolkert) [inline ISSUE_TEMPLATE.md]: I think the extra newlines before and after each comment block aren't necessarily a good use of space.  When I paste it into a new issue, a lot of content ends up below the fold, and removing these newlines is an easy win.

Comment (tvolkert) [reply ISSUE_TEMPLATE.md]: It also increases the likelihood that the user types their content into the comment area by accident.

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,18 +1,60 @@
...
+1. ...
+2. ...
+3. ...
 
 ## Logs
 
-Run your application with `flutter run` and attach all the log output.
+<!--
+
+      Run your application with `flutter run --verbose` and attach all the
```
Comment (tvolkert) [inline ISSUE_TEMPLATE.md]: Many of the issues that are filed are with people unable to get to the point of running.  How about instead:

```
flutter --verbose <command>
```

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,18 +1,60 @@
...
+      exception, please see if the error message includes enough information
+      to explain how to solve the issue.
+
+-->
+
+```
+```
+
+<!--
+
```
Comment (tvolkert) [inline ISSUE_TEMPLATE.md]: Include a section header above for "Analysis"?

File: ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -1,18 +1,60 @@
...
 
-Run `flutter analyze` and attach any output of that command also.
+-->
 
-## Flutter Doctor
+```
+```
 
-Paste the output of running `flutter doctor -v` here.
+<!-- Finally, paste the output of running `flutter doctor -v` here. -->
```
Comment (tvolkert) [inline ISSUE_TEMPLATE.md]: Include a section header above for "Doctor Output"?

Comment (Hixie) [reply ISSUE_TEMPLATE.md]: do the section headers really add much?

**Comment by Hixie** [comment]
Please review https://github.com/flutter/flutter/pull/17213 instead. Thanks.

**Pull request closed without merging**